tools/local-network/local_network.py

```diff
@@ -160,7 +160,7 @@ def __init__(self, name, node_num, client_port, peer_port, admin_port, admin_htt
         self.ledger_distribution_process = None
         self.admin_http_gateway_process = None
         self.ledger_dir = os.path.join(WORK_DIR, f'node-ledger-{self.node_num}')
-        self.ledger_distribution_dir = os.path.join(WORK_DIR, f'node-ledger-distribution-{self.node_num}')
+        self.ledger_distribution_dir = os.path.join(TARGET_DIR, f'node-ledger-distribution-{self.node_num}')
```

Comment:
Why was this changed?

Reply:
Why is that needed? It it a weird hack to only put one of these things over there and not the others.
It also gets in the way of cleaning up past runs by deleting the work dir.
Why not move the entire work dir under target?